packaging/rpm/ondemand-selinux.te

```diff
@@ -77,6 +77,17 @@ allow ood_pun_t self:tcp_socket { create_stream_socket_perms };
 allow ood_pun_t self:udp_socket { create_stream_socket_perms };
 logging_send_syslog_msg(ood_pun_t)
 
+# config
+ifdef(`config_home_t',`
+  type ood_pun_config_t;
+  allow ood_pun_t config_home_t:dir { add_name write create };
+  manage_dirs_pattern(ood_pun_t, config_home_t, config_home_t)
+  manage_files_pattern(ood_pun_t, ood_pun_config_t, ood_pun_config_t)
+  manage_dirs_pattern(ood_pun_t, ood_pun_config_t, ood_pun_config_t) 
+  type_transition ood_pun_t config_home_t:dir ood_pun_config_t;
+  type_transition ood_pun_t ood_pun_config_t:dir ood_pun_config_t;
```

Comment:
Not sure this transition is good, since not all of `config_home_t` is for OnDemand.  The 3 `manage_` you have are likely necessary and same with `allow`.

Reply:
This transition would be necessary if you also updated this to define which paths get this new context:

https://github.com/OSC/ondemand/blob/master/packaging/rpm/ondemand-selinux.fc